### ISSUE TITLE
Consider updating to simdjson 0.2.0

### ISSUE THREAD

**Issue by lemire** [TkTech/pysimdjson#26]
The library simdjson has a new major release (0.2.0). Major changes:

* Support for 64-bit ARM processors, can run under iOS (iPhone).
* Runtime dispatching on x64 processors (switches to SSE on older x64 processors, uses AVX2 when available). Supports processors as far back as the Intel Westmere.
* More accurate number parsing.
* Fixes most warnings under Visual Studio.
* Several small bugs have been fixed.
* Better performance in some cases.
* Introduces a JSON Pointer interface https://tools.ietf.org/html/rfc6901
* Better and more specific error messages (with optional textual descriptions).
* valgrind clean.
* Unified code style (LLVM).

**Comment by TkTech** [comment]
Going to cherrypick some fixes but hold off on updating, seem to be a number of issues.

**Comment by lemire** [comment]
Can you share the nature of the issues?

**Comment by TkTech** [comment]
Nothing in simdjson, just pysimdjson issues with packaging and tests for the new ISAs.

**Comment by TkTech** [comment]
@lemire what are you using for the ARM platform builds? Drone?

**Comment by lemire** [comment]
@TkTech Yes. I also have my own ARM servers. I can give you access if you'd like.

Of course, Amazon offers ARM nodes these days...
